.travis.yml

```diff
@@ -0,0 +1,26 @@
+
+language: ruby
+rvm:
+  - ruby-2.2.3
```

Comment:
Instead of adding this, let's add a `.ruby-version` file. Travis will look at that, but it'll also make it easier to be in the correct Ruby version on the development side.